README.md

```diff
@@ -1 +1,7 @@
-# Hello-Microverse2
\ No newline at end of file
+Hello microverse is the first project in week one
+i made this with html
+![Screenshot from 2021-06-08 12-46-58](https://user-images.githubusercontent.com/75973193/121163891-d53a2680-c857-11eb-9747-f21344944e00.png)
+From the project i learned;
+To use git & github flow
+To setup linters
+To write descritive readme files and pull request descriptions
```

Comment:
- [ ] Please work on the README currently is not descriptive.. don't give up take this as learning because every project you will work on in the future you will be supposed to document it so that any person can understand what is all about. kindly refer to this [Template](https://github.com/microverseinc/readme-template) it will be a good help.